pymc_marketing/mmm/tvp.py

```diff
@@ -99,56 +190,59 @@ def time_varying_prior(
     phi, sqrt_psd = gp.prior_linearized(Xs=X[:, None] - X_mid)
     hsgp_coefs = pm.Normal(f"{name}_hsgp_coefs", dims=hsgp_dims)
     f = phi @ (hsgp_coefs * sqrt_psd).T
+    f = _softplus(f)
     centered_f = f - f.mean(axis=0) + 1
     return pm.Deterministic(name, centered_f, dims=dims)
 
 
-def create_time_varying_intercept(
+def create_time_varying_gp_multiplier(
+    name: str,
+    dims: tuple[str, str] | str,
```

Comment:
I've been importing this from pymc
```python
from pymc.distributions.shape_utils import Dims
```
```suggestion
    dims: Dims,
```

Reply:
Done!